src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 import { ButtonComponent } from '../../common/components';
 import MyStatsComponent from './MyStatsComponent';
 import SocietyStatsComponent from './SocietyStatsComponent';
+import LogACtivityForm from './LogPointsModal';
```

Comment:
The name of this import is a bit misleading importing a form from a modal file

Reply:
@Kachulio1 I have used a more relevant name for the modal